doc/old/image_data_compression.rst

```diff
@@ -151,3 +114,59 @@ is RLE compressed. It's up to you to ensure that the the correct *Photometric
 Interpretation* has been set and that the decompressed pixel data is in the
 correct color space prior to actually calling
 :meth:`~pydicom.dataset.Dataset.compress`.
+
+
+Compressing using third-party packages
+--------------------------------------
+
+If you need to perform pixel data compression using an encoding method not
+supported by *pydicom* - such as JPEG - then you'll need to find a third-party
+package or application to do so. Once you've done that you have to follow the
```

Comment:
Is there a word for 'plain' JPEG? -- some might read this pydicom not supporting any JPEG variants.

Reply:
I can stick the ISO documentation number there, or add an explicit reference to it not being JPEG-LS or J2K. JPEG really has far too many formats.

Reply:
Maybe just leave out the example of 'JPEG' completely?  And maybe link 'support by pydicom' to the supported page.

Reply:
Added the ISO/IEC number plus a link to the DICOM standard

Reply:
The encoding formats supported by pydicom is in the table in the section directly above